backend/src/main/java/com/woowacourse/kkogkkog/config/InterceptorConfig.java

```diff
@@ -20,6 +20,7 @@ public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(new LoginInterceptor(jwtTokenProvider))
             .addPathPatterns("/api/members/me")
             .addPathPatterns("/api/coupons")
-            .addPathPatterns("/api/coupons/*/event");
+            .addPathPatterns("/api/coupons/*/event")
+            .addPathPatterns("/api/coupons/*/event/*");
```

Comment:
이 부분 볼 때마다 느끼는건데 차라리 세미콜론을 아랫줄로 옮기는건 어떨까 싶네요.
 
```suggestion
            .addPathPatterns("/api/coupons/*/event")
            .addPathPatterns("/api/coupons/*/event/*")
            ;
```